hazelcast/src/test/java/com/hazelcast/client/config/YamlClientConfigBuilderResolutionTest.java

```diff
@@ -198,8 +198,46 @@ public void testResolveFromWorkDir() throws Exception {
     }
 
     @Test
-    public void testResolveFromClasspath() throws Exception {
+    public void testResolveFromWorkDirYmlButNotYaml() throws Exception {
+        helper.givenYmlClientConfigFileInWorkDir("cluster-yml-workdir");
+
+        ClientConfig config = new YamlClientConfigBuilder().build();
+
+        assertEquals("cluster-yml-workdir", config.getInstanceName());
+    }
+
+    @Test
+    public void testResolveFromWorkDirYamlAndYml() throws Exception {
+        helper.givenYamlClientConfigFileInWorkDir("cluster-yaml-workdir");
+        helper.givenYmlClientConfigFileInWorkDir("cluster-yaml-workdir");
```

Comment:
Shouldn't this be `cluster-yml-workdir`?

Reply:
I don't think so; I think the purpose is to test the presence of both `.yaml` and `.yml` files in the same directory.

Reply:
Good spot. Will fix.